website/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2'
+
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/etc/nginx/vhost.d"
+      - "/usr/share/nginx/html"
+      - "/var/run/docker.sock:/tmp/docker.sock:ro"
+      - "/etc/nginx/certs"
+    environment:
+      ENABLE_IPV6: "false" # Enable when ready
+
+  letsencrypt-nginx-proxy-companion:
+    image: jrcs/letsencrypt-nginx-proxy-companion
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+    volumes_from:
+      - "nginx-proxy"
+    environment: 
+      ACME_CA_URI: https://acme-staging.api.letsencrypt.org/directory # testing value
+
+  jekyll:
+    command: jekyll serve
+    image: jekyll/jekyll:latest
+    depends_on:
+      - nginx-proxy
+      - letsencrypt-nginx-proxy-companion
+    volumes:
+      - $PWD:/srv/jekyll
+      - $PWD/vendor/bundle:/usr/local/bundle
+    ports:
+      - "4000:4000"
+      - "35729:35729"
+      - "3000:3000"
+    environment:
+      VIRTUAL_HOST: "tomesh.dev,www.tomesh.dev"
+      VIRTUAL_PORT: 4000
+      LETSENCRYPT_HOST: "tomesh.dev,www.tomesh.dev"
+      LETSENCRYPT_EMAIL: "admin@admin.com"
```

Comment:
What email shall we use for this? Shall we make one for tools? https://github.com/tomeshnet/documents/blob/master/service_setup/email.md#public-accounts

Reply:
yes